src/app/functions/server/getExperiments.ts

```diff
@@ -103,20 +89,8 @@ export async function getExperiments(params: {
       throw new Error(`Cirrus request failed: ${response.statusText}`);
     }
 
-    // With the `CirrusV2` flag enabled, the response is `ExperimentData`,
-    // otherwise, it's just `ExperimentData["Features"]`.
-    const json = (await response.json()) as
-      | ExperimentData
-      | ExperimentData["Features"];
-
-    let experimentData: ExperimentData_V2_Or_V2LikeV1;
-    if (flags.includes("CirrusV2")) {
-      experimentData = json as ExperimentData;
-    } else {
-      experimentData = {
-        Features: json as ExperimentData["Features"],
-      };
-    }
+    const json = (await response.json()) as ExperimentData;
+    const experimentData = json;
```

Comment:
suggest simplifying by removing reassignment

```suggestion
    const experimentData = (await response.json()) as ExperimentData;
```